test/test_suite_test.go

```diff
@@ -152,10 +152,12 @@ func reportCreateVMFailure(vm string, err error) {
 var _ = BeforeAll(func() {
 	var err error
 
-	if !config.CiliumTestConfig.Reprovision {
-		// The developer has explicitly told us that they don't care
-		// about updating Cilium inside the guest, so skip setup below.
-		return
+	logger := log.WithFields(logrus.Fields{"testName": "BeforeSuite"})
+	scope, err := helpers.GetScope()
+	if err != nil {
```

Comment:
Quick one, the lines from 172 to 176 should be removed no?
https://github.com/cilium/cilium/blob/205b1ca5f44efae964c4fca561f615c3b739d5ae/test/test_suite_test.go#L155-L176